src/componenet/fetchquotes.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react';
+
+function QuoteComponent() {
```

Comment:
- [OPTIONAL]  Instead of adding component to your component names like xComponent, i suggest you just call the component x as long as its in the component folder. ie **Quote** instead of **QuoteComponent** 👍🏾 

- [ ] Please also name the file the component name. for example quotes.js to have the Quotes component 👍🏾